README.md

```diff
@@ -172,6 +178,134 @@ The OSIsoft connector now has the ability to support OMF. To enable OMF follow t
+      "TenantId":"4da89f7d-b3ce-48a6-b906-b96c367000a9"
+   },
+   "eWONConfig":{
+      "CertificatePath":"/usr"
+   },
+   "AppConfig": {
+       "LoggingLevel": 4,
+       "CycleTimeMs": 1000,
+       "CommunicationType": "omfOcs",
+       "PostDuplicateTagValues": false
+       }
+   }
+   ```
+   1. Upload the updated ConnectorConfig.json file to the /usr directory of your Flexy.
+
+### Flexy OCS BASIC Script
+1. Create a tag in your Flexy for the Basic script to store information.
+   1. On the Flexy's web page, click to expand the "Tags" section.
+   1. Click the "Values" tab.
+   1. Click the "MODE" slider to say "setup".
+   1. Click the "add" button.
+   1. Enter the tag name "tokenReq".
+   1. Under the "I/O Server Setup" section, change the type to "Integer".
+   1. Leave everything else as the default values and click "Add Tag" at the bottom right of the window.
+
+1. Obtain URL encoded client Id and client secret.
+   1. In a web browser such as Internet Explorer or Chrome, navigate to the website https://www.urlencoder.org/
+   1. Copy your client ID into the encoder section of that website.
+   1. Leave the destination character set as UTF-8.
+   1. Leave the destination newline seperator as LF (Unix).
+   1. Hit Encode.
+   1. Copy out the encoded client id and save for later use.
+   1. Copy your client secret into the encoder section of that website.
+   1. Leave the destination character set as UTF-8.
+   1. Leave the destination newline seperator as LF (Unix).
+   1. Hit Encode.
+   1. Copy out your encoded client secret for later use.
+
+
+1. Import the below basic script to the Flexy Basic IDE.
+   1. On the Flexy's web page, click to expand the "Setup" section.
+   1. Click on the "BASIC IDE" tab.
+   1. Click "File".
+   1. Click "Import".
+   1. Click "Select".
+   1. Navigate to the unzipped directory of the OSIsoft Connector and select the file "OcsBasicScript.txt"
+   1. Click "Import".
+   1. Make sure the init section of your Basic IDE looks like the below code snippet.
+   1. Change the xxxxxx for client ID to be your URL encoded client id that you saved previously.
+   1. Change the yyyyyy for client secret your URL encoded client secret you saved previously.
+   1. Click on "File".
+   1. Click on "Save".
+   1. Click on the script execution slider that says "Stopped" to switch it to say "Running".
+   
+```
+// Timer 60 seconds * 30 minutes
+TSET 1, (60 * 30)
+ONTIMER 1, "@updateToken()"
+END
+
+Function updateToken()
+  URL$ = "https://dat-b.osisoft.com/identity/connect/token"
+  Header$ = ""
+  ClientId$ = "6126b79f-458a-49e9-b306-767d1ee1a1cb"
+  ClientSecret$ = "6JafF8HtDp3RULj9SGP4l7eoAvP2%2FcwHQaniB0G5nqw%3D"
+  DataToSend$ = "client_id=" + ClientId$ + "&client_secret=" + ClientSecret$ + "&grant_type=client_credentials"
+  File$ = "/usr/token.json"
+  REQUESTHTTPX URL$, "POST", Header$, DataToSend$, "", File$
+
+  // change tag value here to mark as set
+  tokenReq@ = (tokenReq@ + 1)
+ENDFN
+```
```

Comment:
why is this still here since the readme has the instructions to import the file?

Reply:
I figured this would be a good check since people get easily confused.

Reply:
I guess you're right and it's just clutter now. I can remove this